Cargo.toml

```diff
@@ -14,7 +14,13 @@ anyhow = "1.0"
 clap = { version = "3.1", features = ["derive"] }
 fuel-crypto = "0.4"
 fuel-types = "0.3"
+fuels = { git = "https://github.com/fuellabs/fuels-rs", branch = "master" }
```

Comment:
Temporarily tagging `master` branch because the features needed by this PR haven't been released yet.

Reply:
In the interest of not having things break, can we hold off on this PR until the feature is released?

Reply:
Yup, that's the plan!